src/offline.js

```diff
@@ -1,12 +1,53 @@
 const PouchDB = require('pouchdb')
 
-export function init (cozy, { options = {}, docTypes = [] }) {
-  for (let docType of docTypes) {
-    addDocType(cozy, docType, options)
+export function init (cozy, { options = {}, doctypes = [] }) {
+  for (let doctype of doctypes) {
+    createDatabase(cozy, doctype, options)
   }
 }
 
-export function addDocType (cozy, docType, options) {
+export function createDatabase (cozy, doctype, options = {}) {
   cozy._offline_databases = cozy._offline_databases || []
-  cozy._offline_databases[docType] = new PouchDB(docType, options)
+  cozy._offline_databases[doctype] = new PouchDB(doctype, options)
+  return cozy._offline_databases[doctype]
+}
+
+export function hasDatabase (cozy, doctype) {
+  return cozy._offline_databases && doctype in cozy._offline_databases
+}
+
+export function getDatabase (cozy, doctype) {
+  if (hasDatabase(cozy, doctype)) {
+    return cozy._offline_databases[doctype]
+  }
+  return false
```

Comment:
Why `getDatabase` returns a pouchDB instance (`Object`) or `false`. I know JavaScript allows weak typing, but shouldn't we return consistant JavaScript types like `Object || undefined`.